src/main/java/com/cebanova/gitworkflow/GitWorkflow.java

```diff
@@ -0,0 +1,40 @@
+package com.cebanova.gitworkflow;
+
+public class GitWorkflow {
+    public static void main(String[] args) {
+        // Declaring a list of String variables
+        String step1 = "Perform git stash - to store the uncommitted changes";
+        String step2 = "Perform git checkout master - to move to the local master branch";
+        String step3 = "Perform git fetch command - to fetch the origin branches";
+        String step4 = "Perform checkout -b new_branch_name command - to create and instantly move to a new branch," +
```

Comment:
Please add the step with git pull before performing checkout

Reply:
I have added step 2 . Please review it and let me know if any additional changes are needed.